conference-yaml-schema.yml

```diff
@@ -0,0 +1,123 @@
+$schema: http://json-schema.org/draft-07/schema#
+description: Validation schema for CCFDDL Conference Specification 3.0.X.
+type: array
+items:
+  type: object
+  required: 
+    - title
+    - description
+    - sub
+    - rank
+    - dblp
+    - confs
+  properties:
+    title:
+      description: Short conference name, without year, uppercase
+      type: string
+    description:
+      description: Description, or long name, with no session
+      type: string
+    sub:
+      description: The category that the conference is labeled by CCF.
+      type: string
+      enum:
+        - DS
+        - NW
+        - SC
+        - SE
+        - DB
+        - CT
+        - CG
+        - AI
+        - HI
+        - MX
+    rank:
+      description: The level that the conference is ranked by CCF, e.g., A, B, C
+      type: string
+      enum:
+        - A
+        - B
+        - C
+    dblp:
+      description: The suffix in dblp url, e.g., iccv in dblp.uni-trier.de/db/conf/iccv
+      type: string
+    confs:
+      type: array
+      items:
+        type: object
+        required:
+          - year
+          - id
+          - link
+          - timeline
+          - timezone
+          - date
+          - place
+        properties:
+          year:
+            description: Year the conference is happening
+            type: number
+          id:
+            description: conference name & year, lowercase
+            type: string
+            pattern: ^[a-z0-9-'&]+$
+          link:
+            description: URL to the conference home page
+            type: string
+            format: uri
+          timeline:
+            type: array
+            items:
+              type: object
+              required:
+                - deadline
+              properties:
+                abstract_deadline:
+                  description: Abstract deadline if applicable, optional
+                  type: string
+                  format: date-time
+                  pattern: ^(\d{4}-\d{1,2}-\d{1,2} \d{1,2}:\d{1,2}:\d{1,2}|TBD)$
+                deadline:
+                  description: Deadline, in the format of yyyy-mm-dd hh:mm:ss or TBD
+                  type: string
+                  format: date-time
+                  pattern: ^(\d{4}-\d{1,2}-\d{1,2} \d{1,2}:\d{1,2}:\d{1,2}|TBD)$
+                comment:
+                  description: Some comments on the conference, optional
+                  type: string
+          timezone:
+            type: string
+            enum: 
+              - UTC-12
+              - UTC-11
+              - UTC-10
+              - UTC-9
+              - UTC-8
+              - UTC-7
+              - UTC-6
+              - UTC-5
+              - UTC-4
+              - UTC-3
+              - UTC-2
+              - UTC-1
```

Comment:
```suggestion
              - UTC-1
              - UTC-0
```

Reply:
According to https://github.com/ccfddl/ccf-deadlines/pull/77, there are only `UTC` and `UTC+0` in our data now.